ignite/metrics/accuracy.py

```diff
@@ -125,17 +117,30 @@ def thresholded_output_transform(output):
         device (str or torch.device): specifies which device updates are accumulated on. Setting the metric's
             device to be the same as your ``update`` arguments ensures the ``update`` method is non-blocking. By
             default, CPU.
+        mode (Mode): specifies in which form input will be passed. This can be useful to directly compute
+            accuracy on the output of a neural network, which ofter return probabilities. By default, LABELS.
+        binarization_threshold (float): threshold for binarization of the input, in case a Mode that uses
+            binarization is used.
 
     """
 
+    class Mode(enum.Enum):
+        LABELS = 0
+        PROBABILITIES = 1
+        LOGITS = 2
+
     def __init__(
         self,
         output_transform: Callable = lambda x: x,
         is_multilabel: bool = False,
         device: Union[str, torch.device] = torch.device("cpu"),
+        mode: Mode = Mode.LABELS,
```

Comment:
I'm not sure about correctness of name `LABELS` for multi-class case where we require to pass probas or logits and then take argmax.

Reply:
What about `UNCHANGED` or `RAW_INPUT`?

Reply:
Yes, something like `RAW_INPUT` could work. I'm thinking if we could not generalize this new options to all possible inputs and metric type: binary, multiclass, multilabel...